app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -308,6 +308,10 @@ class HtmlEditorState extends State<HtmlEditor> {
   void initState() {
     super.initState();
     updateEditorState(widget.editorState ?? EditorState.blank());
+    AppFlowyRichTextKeys.partialSliced.addAll([
```

Comment:
isn't that a global change on a singleton? Therefore, should that really be happening everytime we construct a new state? rather than somewhere globally?